app/views/user_sessions/new.html.erb

```diff
@@ -4,6 +4,8 @@
   <%= f.field(:verify, content: recaptcha_tags) if captcha_required? %>
   <div class="submit-buttons">
     <%= f.submit(:login, class: "btn btn-primary") %><br/><br/>
+  </div>
+  <div class="d-flex justify-content-end mt-3">
     <%= link_to(t("user.forgot_password"), new_password_reset_path) unless offline_mode? %>
   </div>
```

Comment:
(Moved to its own div because the old way was a bit hacky; this renders the same as it used to)